src/Compilers/CSharp/Test/Syntax/Diagnostics/DiagnosticTest.cs

```diff
@@ -415,6 +416,7 @@ public void NullableWarnings()
                     ErrorCode.WRN_RecordNamedDisallowed,
                     ErrorCode.WRN_RecordEqualsWithoutGetHashCode,
                     ErrorCode.WRN_AnalyzerReferencesFramework,
+                    ErrorCode.WRN_UnusedRecordParameter,
```

Comment:
>ErrorCode.WRN_UnusedRecordParameter, [](start = 20, length = 36)

It looks like the warning is in the C#9 range. Why do we need to add it to this array?

Reply:
This test catches any warnings in the range 8600 to 9000. The new warning is  8907...
I could adjust the range instead, but figured it is useful to remind us to think about whether the new diagnostics should be considered "nullability" diagnostics.


---
In reply to: [512825729](https://github.com/dotnet/roslyn/pull/48895#discussion_r512825729) [](ancestors = 512825729)